boot/pod/src/boot/file.clj

```diff
@@ -21,7 +21,7 @@
 (def ^:dynamic *sync-delete* true)
 (def ^:dynamic *hard-link*   true)
 
-(def windows? (boot.App/isWindows))
+(def ^:dynamic windows? (boot.App/isWindows))
```

Comment:
This is the change I was most unsure about.  The only reason for `windows?` to be dynamic is for testing purposes, and given that's the case, I didn't think that advertising its dynamic-ness with asterisks was a great idea.

At the time, I didn't know that CI ran on Windows as well, and that fact makes me think there's a better way to do this test? https://github.com/boot-clj/boot/pull/567/files#diff-b6e908691c9bd8d211924e12cfbbcd61R46